### ISSUE TITLE
Mice can eat mispredicted foods inside of a plant bag after opening the plant bag

### ISSUE THREAD

**Issue by GreyMario** [space-wizards/space-station-14#24060]
## Description
**TL;DR: Mice can open inventories, PERIOD, and that's probably the real bug here. They don't get shown the inventory grid, but if the stars align somehow they can still get at what's inside.**

HOO BOY THIS ONE'S A WEIRD ONE

You know how sometimes you end up with plant bags mistakenly showing their contents in-world because of weird shit? Naturally you can't actually pick up or interact with the items because legally they're inside of the plant bag, but this becomes possible once the plant bag is opened.

Now, if you're a mouse... you can't see inside inventories, but you still have the ability to _open_ them by using them or via the context menu. And if it's a plant bag, and the foods inside are "outside" due to jank, you can eat them--but only if the bag is "opened" by you.

_Magic!_

**Reproduction**
Be a mouse.
See a plant bag that's glitched out.
Go try to eat the contents. Fail.
Open the bag. Don't get shown an inventory.
Try to eat the contents, with the bag open. Succeed.
